chimney/src/main/scala/io/scalaland/chimney/dsl/TransformerDefinition.scala

```diff
@@ -115,6 +115,19 @@ final class TransformerDefinition[From, To, C <: TransformerCfg, Flags <: Transf
   def withCoproductInstance[Inst](f: Inst => To): TransformerDefinition[From, To, _ <: TransformerCfg, Flags] =
     macro TransformerDefinitionWhiteboxMacros.withCoproductInstanceImpl[From, To, Inst, C]
 
+  /** Map exact value of a coproduct `from` to a coproduct `to`
+    *
+    * By default if mapping one coproduct in `From` into another coproduct in `To` derivation
+    * expects that coproducts to have matching names of its components, and for every component
+    * in `To` field's type there is matching component in `From` type. If some component is missing
+    * it fails compilation unless provided replacement with this operation.
+    *
+    * @see [[https://scalalandio.github.io/chimney/transformers/customizing-transformers.html#transforming-coproducts]] for more details
+    * @return [[io.scalaland.chimney.dsl.TransformerDefinition]]
+    */
+  def withCoproductInstance[Inst](from: Inst, to: To): TransformerDefinition[From, To, _ <: TransformerCfg, Flags] =
```

Comment:
we need to add such a method to `TransformerInto` as well

Reply:
Thanks, will look into it later. Haven't seen that code before